Firestore/Example/Tests/Local/FSTEagerGarbageCollectorTests.mm

```diff
@@ -35,13 +43,13 @@ - (void)testAddOrRemoveReferences {
   FSTReferenceSet *referenceSet = [[FSTReferenceSet alloc] init];
   [gc addGarbageSource:referenceSet];
 
-  FSTDocumentKey *key = FSTTestDocKey(@"foo/bar");
+  DocumentKey key = testutil::Key("foo/bar");
   [referenceSet addReferenceToKey:key forID:1];
-  FSTAssertEqualSets([gc collectGarbage], @[]);
+  XCTAssertEqual([gc collectGarbage], std::set<DocumentKey>({}));
```

Comment:
Nit: empty curly braces are redundant. Both `std::set<DocumentKey>()`and `std::set<DocumentKey>{}` would work. However, I guess it can be argued that it's more symmetrical with other calls to initializer list constructors, so feel free to keep as is. (Also, this could just check for emptiness... Entirely optional)

Reply:
Ack. I don't know exactly what is the cause in this objc++ world; but using `std::set<DocumentKey>()` will cause `Illegal initializer (only variables can be initialized)` while using `std::set<DocumentKey>{key1, key2}` will cause `Expected unqualified-id`. So the only consistent way is to use both `{` and `(`.

Reply:
`XCTAssertEqual` is a macro that isn't properly hygienic with its arguments. I have seen that sometimes you need to surround the entire argument with a parenthesis.

Something like this may work:

```objc
XCTAssertEqual([gc collectGarbage], (std::set<DocumentKey>{}));
```